benchmark/common.js

```diff
@@ -52,12 +52,13 @@ Benchmark.prototype._parseArgs = function(argv, configs) {
       // Infer the type from the config object and parse accordingly
       const isNumber = typeof configs[match[1]][0] === 'number';
       const value = isNumber ? +match[2] : match[2];
-      cliOptions[match[1]] = [value];
+      if (!cliOptions[match[1]]) cliOptions[match[1]] = [];
```

Comment:
>Can we assign match[1] to a variable

Sure, if you suggest a variable name.

>make the block of this conditional indented on the following line instead?

I'd then have to wrap it in braces, causing this simple default-assignment to take up three lines. But if that's how it's usually done in node's code, then so be it.